### ISSUE TITLE
Associate the video file with the transcription and subtitling

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/meet#1089]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**
When you are on the recording download page, only the video with the audio track is downloadable.
If I am deaf, I cannot understand the meeting only with the video and sound.

**Describe the solution you'd like**
If the meeting transcription has been enabled, it would be useful to include the link to the relevant Doc.

Eventually, when subtitling is available, it would be good to include the subtitle file for download as well, alongside the video.

The video and subtitle files should have the same name with the extension corresponding to the format so that the video launches with the subtitles to avoid an additional step for deaf and hard of hearing users.
